package.json

```diff
@@ -19,14 +19,12 @@
     "publish": "rimraf public_html && cp -r dist public_html",
     "serve-static": "ws -d dist/ -s index.html -p 4242",
     "start": "mkdir -p dist && cp res/zee-worker.js dist/ && NODE_ENV=development node server.js",
-    "start-no-hot": "npm run start",
-    "start-prod": "npm run build-prod && npm run serve-static",
-    "start-prod-readable": "npm run build-prod-readable && npm run serve-static",
-    "test": "npm run build && npm run test-all",
-    "test:quiet": "npm run build:quiet && npm run test-all",
-    "test-all": "NODE_ENV=test nyc mocha --compilers js:babel-core/register 'src/test/**/*.js'",
-    "test-file": "NODE_ENV=test mocha --compilers js:babel-core/register",
-    "test-report": "nyc report --reporter=text-lcov > coverage.lcov"
+    "start-no-hot": "npm run start --",
```

Comment:
This isn't actually new to this patch. I added it on all `npm` calls, because this way we can easily add more arguments.

For example: `npm run start-prod -- --some-argument` translates to `npm run build-prod && npm run serve-static -- --some-argument`. Before it was translating to `npm run build-prod && npm run serve-static --some-argument` which didn't work because `--` was missing.

I wanted it especially for `test:quiet` so that it can pass arguments to the test runner.

Hope this is clear :)